daemon/api.go

```diff
@@ -1189,6 +1190,14 @@ func postSnap(c *Command, r *http.Request, user *auth.UserState) Response {
 	}
 	inst.ctx = r.Context()
 
+	if inst.Channel != "" {
+		ch, err := channel.Parse(inst.Channel, "-")
+		if err != nil {
+			return BadRequest("%v", err)
```

Comment:
do we need/want a special error for ///stable/// ? if not #7670 is probably fine

Reply:
I think it's clear enough as is, and isn't going to be hit by any client we know about